src/panels/RemoveSecretPanel/RemoveSecretPanel.tsx

```diff
@@ -64,13 +65,8 @@ const RemoveSecretPanel = () => {
           uri: secretURI,
         },
       ])
-        .then((response: ErrorResults | string) => {
-          let error;
-          if (typeof response === "string") {
-            error = response;
-          } else if (Array.isArray(response.results)) {
-            error = response.results[0]?.error?.message;
-          }
+        .then((response: ErrorResults) => {
+          const error = response.results[0]?.error?.message;
```

Comment:
I was wondering why here we check the first element in the array, however, in `GrantSecretPanel` we check the first instance of an element in the array that has `error` intead?

Reply:
Ah that's a bug in `GrantSecretPanel`. Here we only care about the error in the first element of the returned array, as we're only passing one secret to be removed. In `GrantSecretPanel` we should display all the errors returned by the API as each application that is being granted can have an error. I'll fix that.

Reply:
I've left a fix comment above to handle this case instead of me doing another PR.

Reply:
Thanks for the review! Fixed it.